en/sdk/anysdk.md

```diff
@@ -49,13 +48,13 @@ If the developer needs to integrate User and IAP system, the game server must de
 
 Parameters can be obtained in the following three ways:
 
-- Click **Dashboard** in the AnySDK service panel to enter the [AnySDK Developer Center](http://dev.anysdk.com/) to get the parameters, as pictured above.
-- After opening the AnySDK service in the Cocos Account Center, click **Dashboard** directly to enter the [ANYSDK Developer Admin Background] to get the parameters.
-- Get parameters in the **Configuration Management -> Game Management** of the AnySDK client.
+- Click **Dashboard** in the **AnySDK** service panel to enter the [AnySDK Developer Center](http://dev.anysdk.com/) to get the parameters, as pictured above.
+- After opening the **AnySDK** service in the **Cocos Account Center**, click **Dashboard** directly to enter the **ANYSDK Developer Admin Background** to get the parameters.

```

Comment:
1. ANYSDK -> AnySDK. Sorry, this is my fault.
2. The [AnySDK Developer Admin Background]'s link is http://dev.anysdk.com/

Reply:
fixed.